packages/js-dapi-client/lib/transport/JsonRpcTransport/requestJsonRpc.js

```
@@ -47,17 +46,21 @@ async function requestJsonRpc(protocol, host, port, selfSigned, method, params,
+  // hides undici from bundler static analysis so it isn't pulled into
+  // browser bundles.
   if (typeof process !== 'undefined'
     && process.versions != null
     && process.versions.node != null
     && protocol === 'https'
     && selfSigned) {
-    requestOptions.agent = new https.Agent({
-      rejectUnauthorized: false,
-    });
+    // eslint-disable-next-line no-eval, global-require
+    const { Agent } = eval('require')('undici');
```

Biome (2.4.15)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)